extensions/amp-story/1.0/test/test-amp-story-page.js

```diff
@@ -700,6 +700,50 @@ describes.realWin('amp-story-page', {amp: {extensions}}, (env) => {
     toggleExperiment(win, 'amp-story-page-attachment-ui-v2', false);
   });
 
+  it('should build the inline page attachment UI with dark theme', async () => {
+    toggleExperiment(win, 'amp-story-page-attachment-ui-v2', true);
+
+    const attachmentEl = win.document.createElement(
+      'amp-story-page-attachment'
+    );
+
+    attachmentEl.setAttribute('layout', 'nodisplay');
+    attachmentEl.setAttribute('cta-image', 'nodisplay');
+    attachmentEl.setAttribute('theme', 'dark');
+    element.appendChild(attachmentEl);
+
+    page.buildCallback();
+    await page.layoutCallback();
+    page.setState(PageState.PLAYING);
+
+    const openAttachmentEl = element.querySelector(
+      '.i-amphtml-story-page-open-attachment'
+    );
+
+    expect(
+      openAttachmentEl.querySelector(
+        '.i-amphtml-story-inline-page-attachment-chip-dark-theme'
+      )
+    ).to.exist;
+    expect(
+      openAttachmentEl.querySelector(
+        '.i-amphtml-story-inline-page-attachment-img-dark-theme'
+      )
+    ).to.exist;
+    expect(
+      openAttachmentEl.querySelector(
+        '.i-amphtml-story-inline-page-attachment-arrow-dark-theme'
+      )
+    ).to.exist;
+    expect(
+      openAttachmentEl.querySelector(
+        '.i-amphtml-story-inline-page-attachment-label-dark-theme'
+      )
+    ).to.exist;
```

Comment:
Like I mentioned in the comment above, a visual diff test would work for this since it will be reflected in the UI so it would be good to remove this test here.

Generally though we should only have one expect per unit test.